resdk/analysis/alignment.py

```diff
@@ -1,23 +1,91 @@
-"""Chip Seq analysis."""
+"""Alignment analysis."""
 from __future__ import absolute_import, division, print_function, unicode_literals
 
 from resdk.resources.utils import get_data_id, get_samples
 
 __all__ = ('bowtie2', 'hisat2')
 
 
-def bowtie2(resource, genome):
+def bowtie2(resource, genome, mode=None, speed=None, use_SE=None, discordantly=None, rep_SE=None,
+            I=None, X=None, trim_5=None, trim_3=None, trim_iter=None, trim_nucl=None,
```

Comment:
Don't use capital letters in variable names (i.e. `rep_se` instead of `rep_SE`) and try to use longer form of names (i.e. `minins` instead of `I`). You can find them in tools' manuals, i.e. http://manpages.ubuntu.com/manpages/xenial/man1/bowtie2.1.html

Also try to keep in sync parameters' descriptions with ones in official documentation.